### PR TITLE
Allow custom class to pagination wrapper.

### PR DESCRIPTION
**Pull request by pdarden**
Change 'Boostrap' to 'Bootstrap' and 'Boostrap4' to 'Bootstrap4'.

Write tests for passing custom class to pagination.

### CODE REVIEW

File: lib/kerosene/html/simple.ex [hunk 1]
```diff
@@ -11,4 +11,13 @@ defmodule Kerosene.HTML.Simple do
 
   defp build_html_class(true), do: "active"
   defp build_html_class(_), do: nil
+
+  defp wrapper_opts(additional_class) do
```
Comment (pdarden) [inline lib/kerosene/html/simple.ex]: Do you mean allowing `opts[:class]` and something like`opts[:current_link_class]` for `Kerosene.HTML.paginate` function so we can also customize current page link?

Comment (allyraza) [inline lib/kerosene/html/simple.ex]:
```elixir
defp build_html_class(class_name) when is_binary(class_name) do
    "pagination #{class_name}"
end
defp build_html_class(true), do: "active"
defp build_html_class(_), do: nil
```

Comment (pdarden) [inline lib/kerosene/html/simple.ex]: Ahh, ok. Will use `build_html_class` instead. Thanks for clarifying!

**Comment by pdarden** [comment]
@allyraza Pushed new commit up. Let me know if it needs more edits.

**Comment by allyraza** [comment]
thank you going to merge

**Pull request merged**